staging/src/k8s.io/apimachinery/go.mod

```diff
@@ -38,4 +37,7 @@ require (
 	sigs.k8s.io/yaml v1.2.0
 )
 
-replace k8s.io/apimachinery => ../apimachinery
+replace (
+	google.golang.org/genproto => google.golang.org/genproto v0.0.0-20220107163113-42d7afdf6368
```

Comment:
there should be no non-local replace directives under staging go.mod files when `hack/lint-dependencies.sh` is happy and you've rerun `hack/update-vendor.sh`

Reply:
Thx. The antlr dep bump triggered quite a few transitive version bumps. Updated.